listenbrainz/webserver/views/user_timeline_event_api.py

```diff
@@ -295,6 +295,58 @@ def user_feed(user_name: str):
     }})
 
 
+@user_timeline_event_api_bp.get('/user/<user_name>/feed/events/<event_id>')
+@crossdomain
```

Comment:
Let's add some more validations to the route.
```suggestion
@user_timeline_event_api_bp.get('/user/<string:user_name>/feed/events/<int:event_id>')
```